src/main/webapp/app/shared/service/privacy-statement.service.ts

```diff
@@ -0,0 +1,27 @@
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { PrivacyStatement, PrivacyStatementLanguage } from 'app/entities/privacy-statement.model';
+import { Observable } from 'rxjs';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class PrivacyStatementService {
+    constructor(private http: HttpClient) {}
+
+    getPrivacyStatement(language: PrivacyStatementLanguage): Observable<PrivacyStatement> {
+        return this.http.get(SERVER_API_URL + 'api/privacy-statement', {
+            params: new HttpParams().set('language', language),
+        }) as Observable<PrivacyStatement>;
```

Comment:
```suggestion
        return this.http.get<PrivacyStatement>(SERVER_API_URL + 'api/privacy-statement', {
            params: new HttpParams().set('language', language),
        });
```
I think you can specify the generic parameter directly.